doc/userguide/capture-hardware/ebpf-xdp.rst

```diff
@@ -249,13 +261,26 @@ also use the ``/etc/suricata/ebpf/xdp_filter.bpf`` (in our example TCP offloadin
     bypass: yes
     use-mmap: yes
     ring-size: 200000
+    # Uncomment the following if you are using hardware XDP with
+    # a card like Netronome
+    # use-percpu-hash: no
 
 
 XDP bypass is compatible with AF_PACKET IPS mode. Packets from bypassed flows will be send directly 
-from one card to the second card without going by the kernel network stack.
+from one card to the second card without going by the ker nel network stack.
 
-Setup symmetric hashing on the NIC
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you are using hardware XDP offload you may have to set ``use-percpu-hash`` to false and
+build and install the XDP filter file after setting ``USE_PERCPU_HASH`` to 0.
+
+In the XDP filter file, you can set ``ENCRYPTED_TLS_BYPASS`` to 1 if you want to bypass
+the encrypted TLS 1.2 packetsin the eBPF code. Be aware that this will mean that Suricata will
+be blind on packets on port 443 with the correct pattern.
+
+If you are not using vlan tracking (``vlan.use-for-tracking`` set to false in suricata.yaml) then you have also to set
```

Comment:
"then you also have to set"

and it's a duplicate of the prior chapter?

Reply:
Yes but you need it in both cases.